internal/cluster/phase.go

```diff
@@ -41,17 +42,17 @@ type OperationPhaseOption struct {
 	Value any
 }
 
-func NewOperationPhase(folder string) (*OperationPhase, error) {
+func NewOperationPhase(folder string, kfdTools config.KFDTools) (*OperationPhase, error) {
 	basePath := folder
 
-	vendorPath, err := filepath.Abs(path.Join(basePath, "../", "vendor"))
+	vendorPath, err := filepath.Abs(path.Join(basePath, "../../", "bin"))
 	if err != nil {
-		return &OperationPhase{}, fmt.Errorf("error getting absolute path for vendor folder: %w", err)
+		return &OperationPhase{}, fmt.Errorf("error getting absolute path for tools bin folder: %w", err)
 	}
 
-	kustomizePath := path.Join(vendorPath, "bin", "kustomize")
-	terraformPath := path.Join(vendorPath, "bin", "terraform")
-	kubectlPath := path.Join(vendorPath, "bin", "kubectl")
+	kustomizePath := path.Join(vendorPath, "kustomize", kfdTools.Kustomize, "kustomize")
+	terraformPath := path.Join(vendorPath, "terraform", kfdTools.Terraform, "terraform")
+	kubectlPath := path.Join(vendorPath, "kubectl", kfdTools.Kubectl, "kubectl")
```

Comment:
```suggestion
	kustomizePath := path.Join(binPath, "kustomize", kfdTools.Kustomize, "kustomize")
	terraformPath := path.Join(binPath, "terraform", kfdTools.Terraform, "terraform")
	kubectlPath := path.Join(binPath, "kubectl", kfdTools.Kubectl, "kubectl")
```

Reply:
I can't tell if those paths are correct but they look a bit odd, can you confirm they are the intended ones (repetition in the path seems weird, but maybe it's ok)?

Reply:
citing the issue: Default value proposal: $HOME/.furyctl/bin/dependency/version/<binary>

so the expected dir for a binary is: ~/.furyctl/bin/kubectl/1.23.7/ plus the binary => ~/.furyctl/bin/kubectl/1.23.7/kubectl